mlflow/models/display_utils.py

```diff
@@ -113,39 +114,45 @@ def should_render_agent_eval_template(signature: ModelSignature) -> bool:
     return _is_signature_agent_compatible(signature)
 
 
+def _generate_agent_eval_recipe(model_uri: str) -> str:
+    resources_dir = Path(__file__).parent / "resources"
+    pip_install_command = """%pip install -U databricks-agents
+dbutils.library.restartPython()
+## Run the above in a separate cell ##"""
+    eval_with_synthetic_code = (
+        (resources_dir / "eval_with_synthetic_example.py")
+        .read_text()
+        .replace("{{pipInstall}}", pip_install_command)
+        .replace("{{modelUri}}", model_uri)
+    )
+    eval_with_dataset_code = (
+        (resources_dir / "eval_with_dataset_example.py")
+        .read_text()
+        .replace("{{pipInstall}}", pip_install_command)
+        .replace("{{modelUri}}", model_uri)
+    )
+
+    # Remove the ruff noqa comments.
+    ruff_line = "# ruff: noqa: F821, I001\n"
+    eval_with_synthetic_code = eval_with_synthetic_code.replace(ruff_line, "")
+    eval_with_dataset_code = eval_with_dataset_code.replace(ruff_line, "")
+
+    return (
+        (resources_dir / "agent_evaluation_template.html")
+        .read_text()
+        .replace("{{ eval_with_synthetic_code }}", html.escape(eval_with_synthetic_code))
+        .replace("{{ eval_with_dataset_code }}", html.escape(eval_with_dataset_code))
```

Comment:
```suggestion
        .replace("{{eval_with_synthetic_code}}", html.escape(eval_with_synthetic_code))
        .replace("{{eval_with_dataset_code}}", html.escape(eval_with_dataset_code))
```

can we also update these for consistency?